app_flutter/lib/service/dev_cocoon.dart

```diff
@@ -101,36 +101,6 @@ class DevelopmentCocoonService implements CocoonService {
       ..branch = 'master';
   }
 
-  static const List<String> _stages = <String>[
-    'cirrus',
-    'chromebot',
-    'devicelab',
-    'devicelab_win',
-    'devicelab_ios',
-  ];
-  static const List<int> _stageCount = <int>[
```

Comment:
@Hixie I changed the cocoon fake data to be 120 LUCI tasks (we no longer serve these other tasks), SG?